.github/ISSUE_TEMPLATE.md

```diff
@@ -3,11 +3,11 @@ Before submitting, please read over our commonly reported issues to prevent dupl
 
 ### All common issues
 
-* [common issues](/docs/common-issues)
+* [common issues](/docs/common-issues.md)
```

Comment:
My concern is that in the current [issue template](https://github.com/airbnb/enzyme/issues/new), this link goes to https://github.com/docs/common-issues - this change only makes it go to https://github.com/docs/common-issues.md, which is also incorrect.

I think that none of the links can have leading slashes, because that makes them absolute links on github.com.